functional_testing/Openvcloud/utils/utils.py

```diff
@@ -407,18 +404,19 @@ def get_machine_nodeID(self, machineId):
         machine = ccl.vmachine.get(machineId)
         stackID = machine.stackId
         nodeID = ccl.stack.get(stackID).referenceId
-        return nodeID
+        return int(nodeID)
```

Comment:
check that there is nothing broken or u can remove the int ant put inside the function

Reply:
only used one time in this ```ovc_master_hosted/OVC/a_basic/machine_tests.pytestcases test007_cleanup_vxlans_for_stopped_deleted_vms```